storage_service/storage_service/settings/production.py

```diff
@@ -4,53 +4,45 @@
 from os import environ
 
 from .base import *  # noqa: F401, F403
-from .base import get_env_variable  # To make he linter happy
-
-# Normally you should not import ANYTHING from Django directly
-# into your settings, but ImproperlyConfigured is an exception.
-from django.core.exceptions import ImproperlyConfigured
-
-
-
-
-def get_env_setting(setting):
-    """ Get the environment setting or return exception """
-    try:
-        return environ[setting]
-    except KeyError:
-        error_msg = "Set the %s env variable" % setting
-        raise ImproperlyConfigured(error_msg)
-
```

Comment:
This is just tidying up: the `get_env_setting` function is not be being used and is identical to the `get_env_variable` function defined in `base.py`. The two imports above were only being used by this function so I removed them too.